### DISCUSSION TITLE
Text UI for navigating / managing items

### DISCUSSION THREAD

**Discussion by NiloCK** [jotaen/xit#38] [Show & tell]
Hi folks. A weekend project has doubled in scope to a two weekender, and unsurprisingly it is growing further and further from being done.

https://github.com/nilock/tuido

This [x]it! spec showing up on HN inspired me to get better organized in managing my everyday `YYYY-MM-DD.md` text files, which themselves have always been littered with undone, forgotten `[ ] things`.

**Comment by jotaen** [comment]
🎉 Awesome project – aggregating and managing todo items from various files in one central place will be super useful. Providing this as an interactive CLI tool seems like a simple yet powerful approach.

I’m happy to link your project from [the website](https://xit.jotaen.net/). That’s just an offer so that other users are able to find it more easily. Let me know if or when you want me to do this.

**Reply by NiloCK** [reply]
Thanks! I'll take up the link offer at a later date - need to fix the pagination issue and enable sorting options first. Also I came across a crash yesterday that I'd like to track down and fix.

**Reply by NiloCK** [reply]
You can go ahead and link now if you like. Far from feature complete, but reasonably functional now as a browser and status manager. Doesn't seem to crash that much :)

**Reply by jotaen** [reply]
Cool, I’ve put up the link!
